cmd/git-init/main.go

```diff
@@ -87,16 +91,18 @@ func main() {
 		run(logger, "git", "init")
 	}
 
-	run(logger, "git", "remote", "add", "origin", *url)
-	err = runOrFail(logger, "git", "fetch", "--depth=1", "--recurse-submodules=yes", "origin", *revision)
-	if err != nil {
+	trimedURL := strings.TrimSpace(*url)
+	runOrFail(logger, "git", "remote", "add", "origin", trimedURL)
+	if err = run(logger, "git", "fetch", "--depth=1", "--recurse-submodules=yes", "origin", *revision); err != nil {
```

Comment:
Could we consider using context to handler command hangs? Today we encountered a situation which the command was stuck, and there was a problem with the underlying storage.

Reply:
@zouyee we may, with a default `timeout`, question is then, what would be a "sane" timeout ?